das-dashboard/backend/services/query_services.py

```
@@ -0,0 +1,162 @@
+            )
+
+        return handlers[command_type](command_type, command_text)
+
+    def _get_query_parameters(self, command_type: str, command_text: str) -> Response:
+        return self._post_execution(command_type, command_text)
+
+    def _set_query_parameters(self, command_type: str, command_text: str) -> Response:
+        try:
+            custom_parameters_dict = json.loads(command_text)
+        except json.JSONDecodeError as error:
+            raise CustomValueError(f"Invalid query parameters payload: {error}") from error
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)